src/strategies/latin_hypercube.jl

```diff
@@ -0,0 +1,101 @@
+"""
+LatinHypercube(nGenerations = 1, popSize = 100, nTournament = 2,
+                pTournament = 0.8. interSampleWeight = 1.0,
+                ae_power = 2, periodic_ae = false)
+
+Instantiate  grid-based hyperparameter tuning strategy using the library
+LatinHypercubeSampling.jl. The optimised Latin Hypercube sampling plan is
+created using a genetic based optimization algorithm based on the inverse of the
+Audze-Eglais function.
+The optimization is run for nGenerations. The population size, number of samples
+selected, probability of selection, the inter sample weight of sampling and the
+norm can be choosen. There is also the possibility of using a periodic version
+of the Audze-Eglais which reduces clustering along the boundaries of the
+sampling plan. This feature can be specified by the periodic_ae variable.
+
+### Supported ranges:
+
+A single one-dimensional range or vector of one-dimensioinal ranges
+can be specified. Specifically, in `LatinHypercubeSampling` search,
+the `range` field of a `TunedModel` instance can be:
+
+- A single one-dimensional range - ie, `ParamRange` object - `r`,
+
```

Comment:
I know this is just copied from Grid, but let's improve this by adding after "`r`" the parenthetic "(constructed using the `range` method)."